packages/core/src/utils/getRemainingProps.js

```diff
@@ -0,0 +1,5 @@
+import omit from 'lodash.omit';
+
+export default function getRemainingProps(props, Component) {
```

Comment:
覺得這邊可以直接傳 propTypes 進去就好

```suggestion
export default function getRemainingProps(props, componentPropTypes) {
```